homeassistant/components/climate/broadlink.py

```diff
@@ -0,0 +1,447 @@
+"""
+Support for Chinese wifi thermostats (Floureon, Beok, Beca Energy)
+configuration.yaml
+climate:
```

Comment:
Add a module docstring as in other platforms with a link to the docs page.

Move this to our docs.